topics/docker/slides/additional.md

```diff
@@ -1,6 +1,60 @@
 Weiterführendes
 ===============
 
+Devcontainer
+------------
+
+[devcontainer]
+
+* Bieten eine vorkonfigurierte, reproduzierbare Entwicklungsumgebung mit Docker-Containern.
+* Entwicklungsumgebung funktioniert konsistent auf Windows, macOS und Linux.
+* `devcontainer.json` definiert das Container-Image sowie Erweiterungen und Tools.
+* Nahtlose Integration mit VS Code, GitHub Codespaces (und Intellij IDEs).
```

Comment:
Warum sind Intellij IDEs in Klammern?

Reply:
Sie unterstützen nicht die volle Funktionsfähigkeit von devcontainers. Aber hab gesehen, dass das keine so grossen unterschiede mehr sind. Sinde entfernt